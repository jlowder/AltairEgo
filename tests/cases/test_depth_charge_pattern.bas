10 FOR D=1 TO 3 : PRINT "TRIAL #";D; : GOSUB 100 : PRINT : NEXT D
20 PRINT "DONE"
30 END
100 PRINT "SUBROUTINE"
110 RETURN
