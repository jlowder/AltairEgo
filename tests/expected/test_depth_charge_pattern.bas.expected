Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TRIAL # 1 SUBROUTINE

TRIAL # 2 SUBROUTINE

TRIAL # 3 SUBROUTINE

DONE
OK
